### PR TITLE
BZ1857224 Known Issue - nodes unavailable on reboot

### PR DESCRIPTION
**Pull request by jeana-redhat**
For https://bugzilla.redhat.com/show_bug.cgi?id=1857224

### CODE REVIEW

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -1834,6 +1834,14 @@ In the table below, features are marked with the following statuses:
 [id="ocp-4-5-known-issues"]
 == Known issues
 
+* An issue has been found that intermittently renders nodes unavailable for 
+workloads after the node reboots. Because nodes are rebooted during a cluster
+upgrade or update, upgrades to {product-title} 4.5 are disabled until the issue
+is resolved. Do not force an upgrade to 4.5 or mirror this release into an
+environment. When it is safe to upgrade, the upgrade button will become
```
Comment (jeana-redhat) [inline release_notes/ocp-4-5-release-notes.adoc:1841]: s/upgrade button/*Update* button

**Review by codyhoag — APPROVED** [review]
LGTM!

**Pull request merged**